src/MSBuildWasm.csproj

```diff
@@ -66,9 +52,15 @@
             FinalOutputPath="$(ProjectDepsFilePath)" />
     </ItemGroup>
 </Target>
-  
+<Target Name="IncludeWasmLibs" BeforeTargets="_GetPackageFiles">
```

Comment:
this is kind of dirty, but we get no other hook since we don't have a per-RID inclusion hook for package, and there's no central package files hook yet either.